protocol/contracts/admin/VestingPayments.sol

```diff
@@ -258,20 +233,22 @@ contract VestingPayments is IVestingPayments, PaymentBase, TempleElevatedAccess
         uint40 _releaseTime
     ) private view returns (uint256) {
         if (_schedule.amount == 0) { return 0; }
-        if (_releaseTime <= _schedule.start) { return 0; }
         // below cliff
         if (_releaseTime <= _schedule.cliff) { return 0; }
         // if revoked, return releasable amount at time of revoke
         if (_schedule.revoked) { return revokedAccountsReleasable[_schedule.recipient]; }
 
-        uint256 _elapsed = _getElapsedTime(_schedule.start, _releaseTime, _schedule.duration);
-        uint256 _vested = TempleMath.mulDivRound(_schedule.amount, _elapsed, _schedule.duration, false);
-        return _vested;
+        // cliff is guaranteed to be greater than start from the checks in createSchedules
+        // cap it to the vestion duration. Therefore _releaseTime is always greater than start.
```

Comment:
typo: vesting duration

Reply:
Would you mind updating?

Reply:
Feel free to hijack the pr to fix and get it landed into yours. I won't get the chance to tweak this weekend anyway